packages/leafygreen-provider/src/BaseFontSizeProvider.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+interface BaseFontSizeProviderProps {
+  children: React.ReactNode;
+  baseFontSize: number;
+}
+
+const BaseFontSizeContext = createContext<number>(14);
+
+export function useBaseFontSize() {
+  return useContext(BaseFontSizeContext);
+}
+
+function BaseFontSizeProvider({
```

Comment:
Should this be `TypographyProvider`? I can't imagine this being the last customization we allow.

Reply:
I agree with the name, though I don't think the purpose here is for these to be consumed independent of the LeafyGreenProvider.

Reply:
👍